src/drive/components/FileList.jsx

```diff
@@ -45,13 +45,11 @@ class FileList extends PureComponent {
   }
 
   shouldDisplayLoadMore() {
-    // We're in /recent
     if (!this.props.displayedFolder) return false
-    if (isCordova()) {
-      if (this.props.files.length < FILES_FETCH_LIMIT) return false
+    else if (isCordova()) {
+      if (this.props.files.length < FILES_FETCH_LIMIT) return false // We're in /recent
       return !this.state.hasNoMoreRows
-    }
-    return this.props.files.length < this.props.fileCount
+    } else return this.props.files.length < this.props.fileCount
```

Comment:
Just making the code pathes more explicit.

Reply:
I think it's not a good idea, it adds a level of indentation for nothing, and the "return fast" pattern is well-known.